suite-native/graph/src/components/AxisLabel.tsx

```diff
@@ -17,8 +19,11 @@ const axisLabelStyle = prepareNativeStyle<Pick<AxisLabelProps, 'x'>>((_, { x })
 
 export const AxisLabel = ({ x, value }: AxisLabelProps) => {
     const { applyStyle } = useNativeStyles();
+    const [isDiscreetMode] = useAtom(isDiscreetModeOn);
```

Comment:
I think it could be nice practice to encapsulate these atoms into it's own hooks, because of easier imports (no need to add jotai to package dependencies) and refactors. I can imagine something like this:

```tsx
const useDiscreetMode = () => {
	const [isDiscreetMode, setIsDiscreetMode] = useAtom(isDiscreetModeOn);
	return {
		isDiscreetMode,
		toggleDiscreetMode: () => setIsDiscreetMode(!isDiscreetMode)
}
```